src/lib/REST/Input/Parser/ContentTree/LoadSubtreeRequest.php

```diff
@@ -31,7 +34,32 @@ public function parse(array $data, ParsingDispatcher $parsingDispatcher): LoadSu
             $nodes[] = $parsingDispatcher->parse($node, $node['_media-type']);
         }
 
-        return new LoadSubtreeRequestValue($nodes);
+        /** @var \Ibexa\Contracts\Core\Repository\Values\Content\Query\Criterion|null $filter */
```

Comment:
Given `processCriteriaArray` method returns `?Criterion`, and variable is not manipulated further, this comment can be removed.

```suggestion
```